### PR TITLE
[#3122] Replace lodash.trimend import by lodash v4.17.21

### PR DESCRIPTION
**Pull request by JhontSouth**
Fixes # 3122
#minor

## Description
This PR allows the use of the _lodash_ modules by the main package and not by individual modules.

## Specific Changes
- Removed every individual module of _lodash_ in the _package.json_ files.
- Modified every import of the individual modules of _lodash_.

## Testing
The following images show the build and the tests running succesfully.
image
image

### CODE REVIEW

**Review by ceciliaavila — APPROVED** [review]

**Review by sw-joelmut — COMMENTED** [review]
I think the package-lock.json needs to be updated.
Its showing old information, for example the `@microsoft/recognizers-text-date-time` library shows this (image), but in the library it only has the three first ones.
image

File: JavaScript/packages/recognizers-number-with-unit/package-lock.json [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
+    },
+    "@microsoft/recognizers-text-number": {
+      "version": "file:../recognizers-number",
+      "requires": {
+        "@microsoft/recognizers-text": "file:https:/registry.npmjs.org/@microsoft/recognizers-text/-/recognizers-text-1.3.0.tgz",
+        "bignumber.js": "^7.2.1"
+      },
+      "dependencies": {
+        "@microsoft/recognizers-text": {
+          "version": "file:https:/registry.npmjs.org/@microsoft/recognizers-text/-/recognizers-text-1.3.0.tgz",
```
Comment (ceciliaavila) [inline JavaScript/packages/recognizers-number-with-unit/package-lock.json]: I think this is not correct. It's still using the packages from the registry.

File: JavaScript/packages/recognizers-date-time/package-lock.json [hunk 2]
```diff
@@ -1,17 +1,60 @@
...
+    },
+    "@microsoft/recognizers-text-number": {
+      "version": "file:../recognizers-number",
+      "requires": {
+        "@microsoft/recognizers-text": "file:https:/registry.npmjs.org/@microsoft/recognizers-text/-/recognizers-text-1.3.0.tgz",
+        "bignumber.js": "^7.2.1"
+      },
+      "dependencies": {
+        "@microsoft/recognizers-text": {
+          "version": "file:https:/registry.npmjs.org/@microsoft/recognizers-text/-/recognizers-text-1.3.0.tgz",
```
Comment (ceciliaavila) [inline JavaScript/packages/recognizers-date-time/package-lock.json]: I think this is not correct. It's still getting the packages from the registry.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

**Review by sw-joelmut — APPROVED** [review]

**Review by ceciliaavila — APPROVED** [review]

**Pull request closed without merging**